src/PrintProvider.jsx

```diff
@@ -43,28 +44,46 @@ export default class PrintProvider extends React.PureComponent {
     };
   }
 
-  regPrintable(key, node, isSingle) {
+  regHiddenAll() {
+    document.body.classList.add(s.hiddenAll);
```

Comment:
we should add some description and disclaimer:
- it's working in one moment print snapshot context
- it used to cover all of React Portals, popups and modals and etc.